src/event/log_event.rs

```diff
@@ -243,6 +243,84 @@ impl Serialize for LogEvent {
     }
 }
 
+impl remap::Object for LogEvent {
```

Comment:
Just to check: I don't have to review this part right? It's only moved without any modifications?

Reply:
Yes correct.